src/app/pages/CreateWalletPage/index.tsx

```diff
@@ -134,7 +134,7 @@ export function CreateWalletPage(props: CreateWalletProps) {
           </AlertBox>
           <Box pad={{ vertical: 'medium' }}>
             <CheckBox
-              label={t('createWallet.confirmSaved', 'I saved my mnemonic')}
+              label={t('createWallet.confirmSaved', 'I’ve safely stored my mnemonic')}
```

Comment:
create-wallet.cy.ts `cy.findByLabelText(/saved/).click({ force: true })` needs to be updated